.eslintrc.js

```diff
@@ -5,10 +5,19 @@ module.exports = {
 
   overrides: [
     {
-      files: ['*.test.js'],
+      files: ['*.ts'],
+      extends: ['@metamask/eslint-config-typescript'],
+      rules: {
+        'no-shadow': 'off',
+        '@typescript-eslint/no-shadow': 'error',
+        'node/prefer-global/url': 'off',
```

Comment:
Installing `@types/node` didn't work to get `tsc` to stop complaining about the one use of the `URL` global we have. Not sure what to do here.

Reply:
Welp. Found this issue: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/34960

That's unfortunate. Disabling this rule and using the import seems like a reasonable workaround for now :+1: